string home_folder
string current_folder
string selected_folder
string[] current_folders
string[] supported_file_types
int32 file_count
string[] current_file_list
string[] current_topic_list

float32[] min_max_delay
float32 set_delay
bool pub_transforms
bool create_transforms

bool running






